graylog2-server/src/main/java/org/graylog2/indexer/searches/Searches.java

```diff
@@ -791,4 +830,16 @@ public FieldTypeException(Throwable e) {
         return indices.build();
     }
 
+    private <T extends JestResult> T execute(Action<T> clientRequest) {
```

Comment:
Not helping in this branch right now, but there's a similar method in the `es-http-indices` branch (#3740, see `JestUtils`).

Maybe we can pull it out of the branch and merge it into `es-http-client` before merging both PRs.